app/src/main/java/protect/card_locker/importexport/CatimaExporter.java

```diff
@@ -99,19 +99,20 @@ private void writeCSV(SQLiteDatabase database, OutputStreamWriter output) throws
         CSVPrinter printer = new CSVPrinter(output, CSVFormat.RFC4180);
 
         // Print the version
-        printer.printRecord("2");
+        printer.printRecord("3");
 
         printer.println();
 
         // Print the header for groups
-        printer.printRecord(DBHelper.LoyaltyCardDbGroups.ID);
+        printer.printRecord(DBHelper.LoyaltyCardDbGroups.ID,
+                DBHelper.LoyaltyCardDbGroups.NAME);
```

Comment:
While I may have messed up before and forgotten to export the order once order got added, we should really fix that now and also export the order.